doc/io_streams.rdoc

```diff
@@ -220,67 +243,82 @@ Or by using method IO#seek:
 
 === Line \IO
 
-You can read an \IO stream line-by-line using these methods:
+You can process an \IO stream line-by-line.
 
-- IO#each_line: Passes each line to the block:
+===== \Method <tt>#each_line</tt>
 
-    f = File.new('t.txt')
-    f.each_line {|line| p line }
+Passes each line to the block:
 
-  Output:
+  f = File.new('t.txt')
+  f.each_line {|line| p line }
 
-    "First line\n"
-    "Second line\n"
-    "\n"
-    "Fourth line\n"
-    "Fifth line\n"
+Output:
 
-  The reading may begin mid-line:
+  "First line\n"
+  "Second line\n"
+  "\n"
+  "Fourth line\n"
+  "Fifth line\n"
 
-    f = File.new('t.txt')
-    f.pos = 27
-    f.each_line {|line| p line }
+The reading may begin mid-line:
 
-  Output:
+  f = File.new('t.txt')
+  f.pos = 27
+  f.each_line {|line| p line }
 
-    "rth line\n"
-    "Fifth line\n"
+Output:
 
-- IO#gets (also in Kernel): Returns the next line (which may begin mid-line):
+  "rth line\n"
+  "Fifth line\n"
 
-    f = File.new('t.txt')
-    f.gets      # => "First line\n"
-    f.gets      # => "Second line\n"
-    f.pos = 27
-    f.gets      # => "rth line\n"
-    f.readlines # => ["Fifth line\n"]
-    f.gets      # => nil
+===== \Method <tt>#gets</tt>
+
+Returns the next line (which may begin mid-line); also in Kernel:
+
+  f = File.new('t.txt')
+  f.gets      # => "First line\n"
+  f.gets      # => "Second line\n"
+  f.pos = 27
+  f.gets      # => "rth line\n"
+  f.readlines # => ["Fifth line\n"]
+  f.gets      # => nil
+
+===== \Method <tt>#readline</tt>
+
+Like #gets, but raises an exception at end-of-stream.
 
-- IO#readline (also in Kernel; not in StringIO):
-  Like #gets, but raises an exception at end-of-stream.
+Also in Kernel; not in StringIO.
 
-- IO#readlines (also in Kernel): Returns all remaining lines in an array;
+===== \Method <tt>#readlines</tt>
+
+ Returns all remaining lines in an array;
   may begin mid-line:
 
     f = File.new('t.txt')
     f.pos = 19
     f.readlines # => ["ine\n", "\n", "Fourth line\n", "Fifth line\n"]
     f.readlines # => []
```

Comment:
I think the indentations of these lines need to be adjusted.

Reply:
Done.